database/migrations/2018_01_22_111222_clean_status_tables.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+use Carbon\Carbon;
+use Illuminate\Support\Facades\DB;
+use App\Team;
+
+class CleanStatusTables extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $qts = DB::table('quest_team')->get();
+        $timestamp = Carbon::now()->toDateTimeString();
```

Comment:
Is there a better estimate than the current timestamp?

Reply:
Sure thing, just updated it to 2017-10-30 @ 6pm, which was when the game started.